andela/ah-orcas-frontend
# #162527519 Redesign the UI for Authors Haven

#### What does this PR do?
```This PR merges the updated UI design for Authors Haven branch to develop branch ```
#### Description of Task to be completed?
``` redesign the UI for signup page```
``` redesign the UI for login page```
``` redesign the navbar and footer ```
#### How should this be manually tested?
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
 ```cd ah-orcas-frontend```
- Checkout to the this branch
 ```git checkout ch-setup-sass-162527519```
- Install the required dependencies:
```npm install```
- Run the local development server:
```npm start```
- You should be able to view the ah-orcas-frontend-application on your browser ```
#### What are the relevant pivotal tracker stories?
[#162527519](https://www.pivotaltracker.com/story/show/162527519)
#### Screenshots (if appropriate)
<img width="1085" alt="screen shot 2018-12-10 at 13 11 40" src="https://user-images.githubusercontent.com/11458891/49725632-354ab480-fc7d-11e8-81b4-3d7f77c457d0.png">
<img width="1080" alt="screen shot 2018-12-10 at 13 11 54" src="https://user-images.githubusercontent.com/11458891/49725647-3c71c280-fc7d-11e8-8334-26705eaa89a3.png">

<img width="1275" alt="screen shot 2018-12-10 at 15 30 59" src="https://user-images.githubusercontent.com/11458891/49733600-faec1200-fc92-11e8-97e8-90b6c32c930b.png">


#### Checklist:
